cloud_governance/main/environment_variables.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
                                                                         'nat_gateway_unused',
                                                                         'zombie_snapshots', 'skipped_resources',
                                                                         'monthly_report']
-        self._environment_variables_dict['cost_policies'] = ['cost_explorer', 'cost_over_usage', 'cost_billing_reports']
+        self._environment_variables_dict['cost_policies'] = ['cost_explorer', 'cost_over_usage', 'cost_billing_reports', 'cost_explorer_payer_billings']
```

Comment:
I dont see env variable for user, costcenter

Reply:
No.
I think mam points to Shai's accounts are in the sheet.
If the cost center is in GSheet, I will mark it as Shai as owner.

Reply:
I think that it will be good to have this option, WDYT ?

Reply:
yes good catch.
I added *COST_CENTER_OWNER* this env, so we can give key-value pairs
cost_center: owner